client-common/src/tendermint/client.rs

```diff
@@ -17,4 +18,7 @@ pub trait Client: Send + Sync {
 
     /// Makes `broadcast_tx_sync` call to tendermint
     fn broadcast_transaction(&self, transaction: &[u8]) -> Result<()>;
+
+    /// Get staked-state from the staked stake address
+    fn get_account(&self, staked_state_address: &[u8]) -> Result<StakedState>;
```

Comment:
because client-common uses chain-core, rpc client support high level api , rpc client can use api easily.

Reply:
but say `broadcast_transaction` doesn't take `&TxAux`, but takes `&[u8]` -- any thoughts @devashishdxt ?

Reply:
`broadcast_transaction` takes a byte array because tendermint RPC expects a byte array. This interface is only to wrap tendermint RPC calls in Rust functions and is not intended to perform any domain specific things. I think, `fn query(&self, data: &[u8], path: &str) -> Result<QueryResult>;` (as suggested by @tomtau) is the right approach here. All other domain specific things can be handled in `NetworkOpsClient` implementation.

Reply:
agree, i will move get_account to network ops client.

Reply:
let's keep rpc client only related to tendermint